git/git.go

```diff
@@ -733,15 +733,18 @@ func CloneWithoutFilters(flags CloneFlags, args []string) error {
 	// not working. You can get around that with https://github.com/kr/pty but that
 	// causes difficult issues with passing through Stdin for login prompts
 	// This way is simpler & more practical.
-	filterOverride := ""
+	filterDriverOverride := ""
```

Comment:
Thanks! 👍  https://github.com/github/git-lfs/pull/1382/commits/9aebb9caed58492bc3b2944bc58b4585ded1e5cb

Although "false" is actually a string because I am referring to the bash built in:
http://tldp.org/LDP/abs/html/internal.html 

> **false**: A command that returns an unsuccessful exit status, but does nothing else.